src/spaceranger/spaceranger_count/config.vsh.yaml

```diff
@@ -0,0 +1,183 @@
+name: spaceranger_count
+namespace: spaceranger
+description: Count gene expression and protein expression reads from a single capture area.
+keywords: [spaceranger]
+links:
+  documentation: https://www.10xgenomics.com/support/software/space-ranger/latest/analysis/running-pipelines/space-ranger-count
+authors:
+  - __merge__: /src/authors/jakub_majercik.yaml
+    roles: [ author ]
+argument_groups:
+  - name: Inputs
+    arguments:
+      - type: file
+        name: --transcriptome
+        required: true
+        description: Path of folder containing 10x-compatible reference
+        example: "/path/to/refdata-gex-GRCh38-2020-A"
+
+      - type: file
+        name: --fastqs
+        required: true
+        description: Path to input FASTQ data
+        example: "/path/to/fastq_folder"
+
+      - type: file
+        name: --probe_set
+        required: true
+        description: CSV file specifying the probe set used
+        example: "Visium_Human_Transcriptome_Probe_Set_v2.0_GRCh38-2020-A.csv"
+
+      - type: file
+        name: --cytaimage
+        required: false
+        description: Brightfield image generated by the CytAssist instrument
+        example: "cyta_image.tif"
+
+      - type: file
+        name: --image
+        required: true
```

Comment:
I'm not to familiar with spaceranger, but is `--image` always required (espcially in combination with for example `--cytaimage`)?